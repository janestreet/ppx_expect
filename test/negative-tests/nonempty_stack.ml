let%expect_test _ =
  ignore
    (Ppx_expect_runtime.For_external.push_output_exn ~here:[%here]
     : Ppx_expect_runtime.For_external.Stack_frame.t) [@alert "-ppx_expect_runtime"]
;;
