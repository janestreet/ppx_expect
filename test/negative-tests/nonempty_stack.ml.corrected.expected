let%expect_test _ =
  ignore
    (Ppx_expect_runtime.For_external.push_output_exn ~here:[%here]
     : Ppx_expect_runtime.For_external.Stack_frame.t) [@alert "-ppx_expect_runtime"]
[@@expect.uncaught_exn
  {|
  ("reached end of test before completion of [with_empty_expect_test_output_async] or similar function"
    (stack ("")))
  |}]
;;
